### ISSUE TITLE
Time of Use / Tiered

### ISSUE THREAD

**Issue by Gwouigwoui** [jrfernandes/ontario_energy_board#49]
Hello,

And thanks for that integration! Is there an easy way to switch between time of use rates and tiered rates? The data is there, but I haven't figured out how to do it.

Thanks

**Comment by Gwouigwoui** [comment]
Hey! Any update on this?

**Comment by jrfernandes** [comment]
Hi. Definitely worth adding this. Will take a look when I have a chance, unless someone contributes for this feature.

**Comment by dlutz57** [comment]
I have an issue (which is more of a request), and think it related to this one. 
I am in the hydro one residential R2 zone, which currently supports the following plans.
Time-of-use
Ultra-low-overnight
Tiered

In my area we are only offered the Tiered plan.

I can see in the data feed https://www.oeb.ca/_html/calculator/data/BillData.xml that the tiered rates are there:

```
<ET1>600</ET1>
<RPP1>0.12</RPP1>
<RPP2>0.142</RPP2>
```
ET1 is the amount of power where we switch from RRP1 to RRP2, and that values listed match my hydro bill.

Not sure if these values can be exposed, or if you can muddy the mix/logic by supporting the Tiered plan in addition to the TOU and ULO plans currently supported? 

For me just exposing the values would work, since you would need to know my consumption to know which rate (RPP1 or RPP2) is currently being used.

**Comment by dlutz57** [comment]
Never mind, I see that what I need is already there and exposed, they were just hidden.

```
ET1   is Tier threshold   -  sensor.hydro_one_networks_inc_r2_residential_tier_threshold
RPP1  is Lower Tier Price  -  sensor.hydro_one_networks_inc_r2_residential_lower_tier_price
RPP2  is Higher Tier Price  -  sensor.hydro_one_networks_inc_r2_residential_tier_threshold
```

**Comment by jrfernandes** [comment]
That's right. Part of the recent [v1.0.0](https://github.com/jrfernandes/ontario_energy_board/releases/tag/v1.0.0) update. The README includes a list of all the sensors here: https://github.com/jrfernandes/ontario_energy_board/releases/tag/v1.0.0
